risc0/binfmt/src/image.rs

```diff
@@ -42,6 +42,7 @@ pub struct MemoryImage {
     /// Metadata about the structure of the page table
     pub info: PageTableInfo,
 
+    // TODO(1.0): Include the pc in the MemoryImage digest and remove it from SystemState.
```

Comment:
I don't think we can include the `pc` in the digest unless we make a circuit change, and that's not planned for 1.0.